openassessment/xblock/test/data/submission_unavailable.xml

```diff
@@ -1,9 +1,13 @@
 <openassessment submission_start="4999-04-01">
     <title>Open Assessment Test</title>
-    <prompt>
-        Given the state of the world today, what do you think should be done to
-        combat poverty? Please answer in a short essay of 200-300 words.
-    </prompt>
+    <prompts>
+        <prompt>
+            <description>Given the state of the world today, what do you think should be done to combat poverty?</description>
+        </prompt>
+        <prompt>
+            <description>Given the state of the world today, what do you think should be done to combat pollution?</description>
```

Comment:
Actually, these two prompts are different; the first asks about poverty and the second about pollution. Sorry, couldn't think of an original question. ;-)